src/main.rs

```diff
@@ -70,6 +92,116 @@ impl BFInterp {
             MetadataOperand => todo!(),
         }
     }
+
+    fn load(&mut self, op: &Operand, dest: &name::Name) {
+        match op {
+            LocalOperand { name, .. } => {
+                self.mem.insert(dest.clone(), *self.mem.get(name).unwrap());
+                self.mem.remove(name);
```

Comment:
Why do we remove `name` (aka the source) after loading it?

Reply:
Load in the interpreter is essentially switching the identifier for a value. However, I will take it out as I think the same source value can be used more than once.